Formula/ninja.rb

```diff
@@ -18,15 +19,23 @@ class Ninja < Formula
     sha256 "8070023444b46cc29d7e52b71cdda279c4734d96d29c7785302ae0ffe27b1245" => :high_sierra
   end
 
+  depends_on "cmake" => :build
+  depends_on "re2c"
```

Comment:
Does `ninja` require `re2c` in a runtime?

Reply:
Possibly not, I'll look into it when I get a chance.

Reply:
Hmm, I'm actually not even sure it's even a Homebrew build dependency because the files it generates are committed to the repository.

Reply:
Base on the test, I would say it is a build dep.